lang/lang_en_fr.txt

```diff
@@ -1240,4 +1240,196 @@
 
 #MSG_OFF
 " [off]"
+" [off]"
+
+#
+"Back"
+"Retour"
+
+#
+"Checks"
+"Verifications"
+
+#
+"False triggering"
+"Faux declenchement"
+
+#
+"FINDA:"
+"FINDA:"
+
+#
+"Firmware   [none]"
+"Firmware [aucune]"
+
+#
+"Firmware [strict]"
+"Firmware[stricte]"
+
+#
+"Firmware   [warn]"
+"Firmware  [avert]"
+
+#
+"HW Setup"
+"Config HW"
+
+#
+"IR:"
+"IR:"
+
+#
+"Magnets comp.[N/A]"
+"Comp. aimants[N/A]"
+
+#
+"Magnets comp.[Off]"
+"Comp. aimants[Off]"
+
+#
+"Magnets comp. [On]"
+"Comp. aimants [On]"
+
+#
+"Mesh         [3x3]"
+"\x00"
+
+#
+"Mesh         [7x7]"
+"\x00"
+
+#
+"Mesh bed leveling"
 "\x00"
+
+#
+"MK3S firmware detected on MK3 printer"
+"Firmware MK3S detecte sur imprimante MK3"
+
+#
+"MMU Mode [Normal]"
+"Mode MMU [Normal]"
+
+#
+"MMU Mode[Stealth]"
+"Mode MMU [Furtif]"
+
+#
+"Mode change in progress ..."
+"Changement de mode en cours..."
+
+#
+"Model      [none]"
+"Modele   [aucune]"
+
+#
+"Model    [strict]"
+"Modele  [stricte]"
+
+#
+"Model      [warn]"
+"Modele    [avert]"
+
+#
+"Nozzle d.  [0.25]"
+"Diam. buse [0.25]"
+
+#
+"Nozzle d.  [0.40]"
+"Diam. buse [0.40]"
+
+#
+"Nozzle d.  [0.60]"
+"Diam. buse [0.60]"
+
+#
+"Nozzle     [none]"
+"Buse     [aucune]"
+
+#
+"Nozzle   [strict]"
+"Buse    [stricte]"
+
+#
+"Nozzle     [warn]"
+"Buse      [avert]"
+
+#
+"G-code sliced for a different level. Continue?"
+"\x00"
+
+#
+"G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+"\x00"
+
+#
+"G-code sliced for a different printer type. Continue?"
+"Le G-code a ete prepare pour une autre version de l'imprimante. Continuer?"
+
+#
+"G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+"Le G-code a ete prepare pour une autre version de l'imprimante. Veuillez decouper le modele a nouveau. L'impression a ete annulee. "
+
+#
+"G-code sliced for a newer firmware. Continue?"
+"Le G-code a ete prepare pour une version plus recente du firmware. Continuer?"
+
+#
+"G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+"Le G-code a ete prepare pour une version plus recente du firmware. Veuillez mettre a jour le firmware. L'impression  annulee."
+
+#
+"PINDA:"
+"PINDA:"
+
+#
+"Preheating to cut"
+"Prechauffage pour couper"
+
+#
+"Preheating to eject"
+"Prechauffage pour ejecter"
+
+#
+"Printer nozzle diameter differs from the G-code. Continue?"
+"Diametre de la  buse de l'imprimante different du G-Code. Continuer ?"
+
+#
+"Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+"Diametre de la  buse de l'imprimante different du G-Code. Merci de verifier le parametre dans les reglages. Impression annulee."
+
+#
+"Rename"
+"Renommer"
+
+#
+"Select"
+"Selectionner"
+
+#
+"Sensor info"
+"Info capteur"
+
+#
+"Sheet"
+"Feuille"
+
+#
+"Sound     [assist]"
```

Comment:
After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]